SEO: reword introduction paragraph into descriptions - Part 1 (#408)
front-commerce/developers.front-commerce.com: source/docs/index.md

---
id: introduction
title: Introduction

> description: Grasping all the concepts around a product such as Front-Commerce is exciting, but can be intimidating. We tried to create and organize this documentation in a way that you could quickly find the information you need depending on your context.

Reviewer: ```suggestion
description: Grasping all the concepts around a product such as Front-Commerce is exciting but can be intimidating. Therefore, we tried to create and organize this documentation so that you could quickly find the information you need depending on your context.
```
